app/templates/about.jade

```diff
@@ -436,9 +436,9 @@ block content
           h3(data-i18n="about.contact_title")
           h4(data-i18n="about.contact_subtitle")
           if me.showChinaResourceInfo()
-            p
-            | 商务合作：bill@codecombat.com <br/>
-            | 业务咨询：13810906731
+            p.
```

Comment:
Do we need this `.`?

Reply:
We may need this dot for multiline html text